compiler/infra/Uncopyable.hpp

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2022 IBM Corp. and others
```

Comment:
As this was lifted from FrontEnd.hpp, I think the first copyright date should be 2000.